modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/AggregationFormBase.tsx

```diff
@@ -391,6 +392,7 @@ export function AggregationFormBase({
 						'no-fields-were-found'
 					)}
 					error={errors.objectFieldName}
+					id="aggregation-field"
```

Comment:
Hi @Artartur. Please, rename the id to `objectFieldAggregationField`.

Reply:
Done!